skbio/stats/distance/tests/test_mantel.py

```diff
@@ -59,7 +59,7 @@ class MantelTests(MantelTestData):
     def setUp(self):
         super(MantelTests, self).setUp()
 
-        self.methods = ('pearson', 'spearman')
+        self.methods = ('pearson', 'spearman','kendalltau')
```

Comment:
As per PEP8, there should be a space after the comma before `kendalltau`:

```suggestion
        self.methods = ('pearson', 'spearman', 'kendalltau')
```